source/d3d11/runtime_d3d11.cpp

```diff
@@ -443,7 +446,7 @@ bool reshade::d3d11::runtime_d3d11::init_effect(size_t index)
 			if (_renderer_id < D3D_FEATURE_LEVEL_11_0)
 			{
 				effect.errors += "Compute shaders are not supported in ";
-				effect.errors += "D3D10";
+				effect.errors += "D3D11";
```

Comment:
Found a bug

Reply:
That is actually intentional, see the feature level check above. Compute shaders are only fully supported in feature level 11.0+, but not in feature level 10, hence why it says they are not supported in D3D10 (it's running D3D10 feature level under the D3D11 API).